### PR TITLE
Mobile portfolio2

### PR DESCRIPTION
**Pull request by KayLemba**
In this project I created a cards section and an about me section I also did the following:
1. Add cards section and About me section to index.html file
2. Add new styling to styles.css file
3. Update READ.md file
4. Add new screenshot of workflow

### CODE REVIEW

**Review by Elbie-em — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED 🟡  ♻️

Hi @KayLemba,  you have really done a great job so far, I like how you are coming close to the actual design provided in the requirements, however, there are some issues that you still need to work on in order to have this project approved, but you are almost there! Check my comments below ⬇️  for more details

Keep on with the hard work 💪  , Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Kindly submit another review request once done and please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._


> Elbie Moonga (Part-time TSE)

> Github (@Elbie-em)
> Slack (Elbie Moonga)
> Twitter ([@ElbieEm](https://twitter.com/ElbieEm))
> LinkedIn([Elbie Moonga](https://www.linkedin.com/in/elbiemoonga/))

File: index.html [hunk 1]
```diff
@@ -38,4 +38,198 @@ <h1 class="salutation">
           <li><a href="#"><img src="./images/angellist.png" alt="social"></a></li>
         </ul>
       </div>
+    </section>
+    <section class="cards-section">
```
[the remark this answers is not in the document]
Comment (KayLemba) [reply index.html:42]: Done

File: index.html [hunk 2]
```diff
@@ -38,4 +38,198 @@ <h1 class="salutation">
...
+                    <li>css</li>
+                    <li>javascript</li>
+                  </ul>
+                </div>
+
+                <button type="button" class="btn-one">See project</button>
+              </li>
+            </ul>
+          </div>
+        </li>
```
[the remark this answers is not in the document]
Comment (KayLemba) [reply index.html:74]: Done

File: index.html [hunk 3]
```diff
@@ -38,4 +38,198 @@ <h1 class="salutation">
...
+                  </ul>
+                </div>
+
+                <button class="btn-one">See project</button>
+              </li>
+            </ul>
+          </div>
+        </li>
+      </ul>
+    </section>
```
[the remark this answers is not in the document]
Comment (KayLemba) [reply index.html:161]: done

File: index.html [hunk 4]
```diff
@@ -38,4 +38,198 @@ <h1 class="salutation">
...
+              </li>
+
+              <li>
+
+                <div class="tec"><a href="#">
+                    <h2 class="mark-up"> Skills</h2>
+                  </a><img src="images/right-arrow.png" alt="right-arrow"></div>
+
+              </li>
+            </ul>
```
[the remark this answers is not in the document]
Comment (KayLemba) [reply index.html:231]: done

**Review by barackm — APPROVED** [review]
Hi @KayLemba, 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by KayLemba** [comment]
Thank you, I really had it rough trying to correct this work. I'm glad I got it done.

**Pull request merged**